web/themes/new_weather_theme/templates/partials/alerts-in-hourly-table.html.twig

```diff
@@ -0,0 +1,20 @@
+{% set periodLength = periods | length %}
+{% for alertInfo in alertPeriods %}
+  {% set remainingCount = periodLength - alertInfo.duration - alertInfo.periodIndex %}
+  <tr data-row-name="alert">
+    <th scope="row" class="position-sticky left-0"><span class="usa-sr-only">{{ "Alert" | t }}</span></th>
+    {% if alertInfo.periodIndex > 0 %}      
+      {% for i in 1..alertInfo.periodIndex %}
+        <td></td>
+      {% endfor %}
+    {% endif %}
+    <td colspan="{{alertInfo.duration}}" style="background-color:yellow;">
+      {{alertInfo.alert.event}}
+    </td>
+    {% if remainingCount > 0 %}
+      {% for i in 1..remainingCount%}
+        <td></td>
+      {% endfor %}
```

Comment:
If the other colspan would work, I reckon this one could too?

```suggestion
      <td colspan="{{ remainingCount }}"></td>
```

Reply:
This is making me think we should also create an accessibility testing ticket for this table. I'll run it through a screen reader later because I'm really curious what that experience is like.